.circleci/config.yml

```diff
@@ -258,21 +249,6 @@ jobs:
           command: |
             bash <(curl -s https://codecov.io/bash) -F mixer
 
-  docker:
```

Comment:
Why remove ?

Reply:
This is a duplicate. There is a docker-push as well.